main/acle.rst

```diff
@@ -3088,720 +2438,614 @@ Examples
     return n == 32 ? n : (31-n);
   #endif
   }
+`
 
-16-bit multiplications
-======================
+## 16-bit multiplications
 
 The intrinsics in this section provide direct access to the 16x16 and
 16x32 bit multiplies introduced in Armv5E. Compilers are also
 encouraged to exploit these instructions from C code. These intrinsics
-are available when ``__ARM_FEATURE_DSP`` is defined, and are not
+are available when `__ARM_FEATURE_DSP` is defined, and are not
 available on non-5E targets. These multiplies cannot overflow.
-::
 
-  int32_t __smulbb(int32_t, int32_t);
+  `int32_t __smulbb(int32_t, int32_t);`
 
 Multiplies two 16-bit signed integers, i.e. the low halfwords of the
 operands.
-::
 
-  int32_t __smulbt(int32_t, int32_t);
+  `int32_t __smulbt(int32_t, int32_t);`
 
 Multiplies the low halfword of the first operand and the high halfword
 of the second operand.
-::
 
-  int32_t __smultb(int32_t, int32_t);
+  `int32_t __smultb(int32_t, int32_t);`
```

Comment:
OK, I am tired of marking all the WWT`. :)

You are better off for looking at them yourself. These are the places where you had the ::, then the empty line, then the code